### PR TITLE
support event subscription

### PR DESCRIPTION
**Pull request by czerasz**
## what
* add support for `event_subscription` block

## why
* improve module

## references
* [Terraform documentation: `inspector_assessment_template`](https://registry.terraform.io/providers/hashicorp/aws/4.48.0/docs/resources/inspector_assessment_template)

### CODE REVIEW

**Comment by czerasz** [comment]
@jamengual I see the CI got stuck... Anything I can do to improve this state?

**Comment by czerasz** [comment]
@jamengual I see the required jobs are still stuck :slightly_frowning_face:

File: examples/complete/terraform.tfvars.tmp [hunk 1]
```diff
@@ -0,0 +1,8 @@
+enabled_rules = ["cis"]
```
Comment (joe-niland) [inline examples/complete/terraform.tfvars.tmp]: This file should be removed but you may want to integrate this into the examples/complete test

File: examples/complete/.terraform.lock.hcl [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# This file is maintained automatically by "terraform init".
```
Comment (joe-niland) [inline examples/complete/.terraform.lock.hcl]: Please remove this file

**Review by joe-niland — CHANGES_REQUESTED** [review]
Looks good. Just a couple of small things.

File: examples/complete/terraform.tfvars.tmp (hunk 1 above)
Comment (czerasz) [reply examples/complete/terraform.tfvars.tmp]: Removed!

File: examples/complete/.terraform.lock.hcl (hunk 2 above)
Comment (czerasz) [reply examples/complete/.terraform.lock.hcl]: Removed!

**Review by joe-niland — APPROVED** [review]

**Review by aknysh — APPROVED** [review]

**Comment by joe-niland** [comment]
Thanks @czerasz for your contribution

**Pull request merged**